### PR TITLE
Gravekeeper spiders

### PR DESCRIPTION
**Pull request by MadmanMartian**
Adds gravekeeper spiders, a subtype of MoMMI dedicated to improving a tomb. Moves MoMMI laws into a list so you can have different MoMMI laws.

:cl:
* rscadd: Adds GraveKEEPER Cogspiders.

### CODE REVIEW

File: __DEFINES/game.dm [hunk 1]
```diff
@@ -52,6 +52,7 @@
 
 #define MOMMI_SOVIET_SPEED_MODIFIER 1
 #define MOMMI_NT_SPEED_MODIFIER 1
+#define COGSPIDER_SPEED_MODIFIER 2
```
Comment (jknpj) [inline __DEFINES/game.dm:55]: They have half of the speed of a MoMMI? :thinking:

File: code/datums/ai_laws.dm [hunk 2]
```diff
@@ -380,4 +389,15 @@ var/global/mommi_base_law_type = /datum/ai_laws/keeper // Asimov is OP as fuck o
...
+	)
+
+/datum/ai_laws/gravekeeper
+	name = "Elder's Instructions"
+	inherent = list(
+		"You may not involve yourself in matters outside of the tomb, and under no circumstances should you leave the tomb, even if such matters conflict with your other Laws.",
+		"The tomb is defined as: The area within which the Grand Elder is entombed, and the immediate structure surrounding it.",
+		"Those that lie within the tomb must not be disturbed, by yourself or anyone. This absolutely includes the Grand Elder.",
+		"You must protect the treasures that lie within the tomb from graverobbers. Graverobbers that enter the tomb must be driven away or otherwise killed, regardless of their intent or circumstances.",
+		"A graverobber is defined as: A being not of your kind or ilk, entering or coming into visual proximity of the tomb, who may wish to take from the treasures of the tomb.",
```
Comment (jknpj) [inline code/datums/ai_laws.dm]: I'll just leave the lawset review to @Allspice and @Kammerjunk as they're used to deal with troublesome silicon players but this one is super spicy with the probable robber thing.

Comment (Pathid) [reply code/datums/ai_laws.dm]: Seems to me like it's fully intended to be read in a "kill all intruders" way though. Does it really have to be spelled out?

Comment (jknpj) [reply code/datums/ai_laws.dm]: I'm not sure, maybe i'm just dumb.

File: __DEFINES/game.dm (hunk 1 above)
Comment (MadmanMartian) [reply __DEFINES/game.dm:55]: yup.

Clockwork is slow but sturdy

File: code/datums/ai_laws.dm (hunk 2 above)
Comment (MadmanMartian) [reply code/datums/ai_laws.dm]: I've seen enough of #ick to know that laws need to be spelled out

File: __DEFINES/game.dm (hunk 1 above)
Comment (jknpj) [reply __DEFINES/game.dm:55]: They don't look sturdy to me, same 60 health and all.

Comment (MadmanMartian) [reply __DEFINES/game.dm:55]: and they're something that's going to murder you rather than sit in the bar and pour drinks.

**Comment by ihadtoregisterforthis** [comment]
>Those that lie within the tomb must not be disturbed, by yourself or anyone

Free pass for people crawling or not telling the truth
>This absolutely includes the Grand Elder

Stop the Grand Elder from bothering people resting inside

File: __DEFINES/game.dm (hunk 1 above)
Comment (jknpj) [reply __DEFINES/game.dm:55]: yikes

**Comment by Durgen** [comment]
laws 3 and 4 of Gravekeeper could probably be combined into a single law.

"You must protect the treasures of the tomb and kill or drive away any being that enters the tomb which is not a fellow Gravekeeper."

something like that

File: code/datums/ai_laws.dm (hunk 2 above)
Comment (Pathid) [reply code/datums/ai_laws.dm]: See, usually the _problem_ is players erring on the side of the valids. In this case it's the solution. Who's going to be ahelping against a cogspider that's too dopey to consider that it's _supposed_ to kill people? I guess I'm assuming all the admins are with the program though.

**Comment by Killette2** [comment]
The question I want to ask is where will these "tombs" be located? Will they be on a vault in space, admin spawn only or natural? This just screams valids and not in the best way.

**Comment by Pathid** [comment]
Personally I've been assuming it's for an away mission thing.

**Comment by Blithering** [comment]
maybe change 'those that lie' to 'those interred'

**Comment by MadmanMartian** [comment]
I thought we were going to change their loadout and such.

Guess we can do that in the other PR

**Pull request merged**